### ISSUE TITLE
bug with CQI reporting configuration

### ISSUE THREAD

**Issue by sedcore** [srsLTE/srsUE#15]
Line `https://github.com/srsLTE/srsUE/blob/next/ue/src/upper/rrc.cc#L1099` is wrong.

It should be:
```
      current_cfg->cqi_report_cnfg.report_periodic_setup_present = phy_cnfg->cqi_report_cnfg.report_periodic_setup_present;
```

This bug made srsUE unable to connect to new versions of openairinterface
(post commit 9bf890cc8972ae977483a9fcbe10a0a8971c6ee0, which is
from August 2016), because it includes CQI in PUSCH and openair does not
like it at all.

You may also want to review all the places where you force `present` to `true`,
without taking into account the potential `release` case found in the RRC.

Basically, I would say that instead of setting to `true` you can take the value
coming from `liblte/src/liblte_rrc.cc`, if applicable.

Sorry, I don't have time to do this admittedly tedious job.

I hope I am clear with my explanations. If you don't understand, we can iterate.

**Comment by ismagom** [comment]
Thanks for spotting this. I just fixed it. The bug was only in a couple of places where the setup_present field is outside the structure being copied. Thanks
